src/platform/startup/index.js

```diff
@@ -3,15 +3,11 @@
  * @module platform/startup
  */
 import React from 'react';
-import * as Sentry from '@sentry/browser';
 import { Provider } from 'react-redux';
 import { Router, useRouterHistory, browserHistory } from 'react-router';
 import { createHistory } from 'history';
-import { connectFeatureToggle } from 'platform/utilities/feature-toggles';
-
-import createCommonStore from './store';
-import startSitewideComponents from '../site-wide';
 import startReactApp from './react';
+import setUpCommonFunctionality from './setup';
```

Comment:
The changes in these startup files are:

1. Extract common setup functionality to a separate file
2. Create a new `router` file that starts an app using the newer version of React Router

I didn't want to import both in the same file because that puts both versions in everyone's bundles (despite it seeming like tree-shaking should not allow that).